ui/src/components/home/VisionMissionSection.js

```diff
@@ -0,0 +1,412 @@
+                  <Box
+                    w="100%"
+                    position="absolute"
+                    h="107%"
+                    bgPos="30% 100%"
+                    bgSize="50%"
+                    bgRepeat="no-repeat"
+                    bgImage={`url('${process.env.PUBLIC_URL}/images/home/vector-green-2.svg')`}
+                  ></Box>
+                  <Heading
+                    color="#0C0C20"
+                    fontFamily="DMSerifDisplay_Italic, serif"
+                    fontWeight="400"
+                    lineHeight="1"
+                    wordBreak="break-word"
+                  >
+                    Our Vision<br></br> & Mission
+                  </Heading>
+                </Box>
+              </Box>
+              <Flex>
+                <VStack
+                  borderRadius="0.5rem"
+                  w="49.5%"
+                  border="1px solid"
+                  borderColor="green.400"
+                  p="1em"
+                >
+                  <Box borderRadius="1.875em" bgColor="white" m="auto" px="1em">
+                    <Text
+                      color="#0C0C20"
+                      fontSize="20px"
+                      fontFamily="Manrope"
+                      fontWeight="700"
+                      textTransform="uppercase"
+                      lineHeight="20.6px"
+                      letterSpacing="4px"
+                      wordBreak="break-word"
+                    >
+                      VISION
+                    </Text>
+                  </Box>
+                  <Box>
+                    <Text>
+                      <Text display="inline">
+                        <Text
+                          color="#4A6EEB"
+                          fontSize="20px"
+                          fontFamily="DMSerifDisplay_Italic, serif"
+                          fontWeight="400"
+                          lineHeight="28.84px"
+                          wordBreak="break-word"
+                          display="inline"
+                        >
+                          Multiplying
+                        </Text>{' '}
+                        <Text
+                          color="#141419"
+                          fontSize="20px"
+                          fontFamily="DMSerifDisplay_Regular, serif"
+                          fontWeight="400"
+                          lineHeight="28.84px"
+                          wordBreak="break-word"
+                          display="inline"
+                        >
+                          churches in
+                        </Text>{' '}
+                        <Text
+                          color="#141419"
+                          fontSize="20px"
+                          fontFamily="DMSerifDisplay_Italic, serif"
+                          fontWeight="400"
+                          lineHeight="28.84px"
+                          wordBreak="break-word"
+                          display="inline"
```

Comment:
You can actually do 

```
<Text
as="span"
>
....
</Text>

```

This will allow the parent element to directly handle it as an individual text. So you don't need to put display "inline" If you are finding that it doesn't render spaces properly, you can put

`whiteSpace="pre-wrap"` to make it follow your whitespaces,